support/cas-server-support-redis-core/src/main/java/org/apereo/cas/redis/core/RedisObjectFactory.java

```diff
@@ -56,70 +61,72 @@
      * @return the redis connection factory
      */
     public static RedisConnectionFactory newRedisConnectionFactory(final BaseRedisProperties redis) {
-        val poolConfig = redis.getPool() != null
-            ? redisPoolConfig(redis)
-            : LettucePoolingClientConfiguration.defaultConfiguration();
+        LOGGER.debug("Loaded Redis configuration {}", redis);
 
-        val sentinelConfiguration = redis.getSentinel() == null
-            ? null
-            : potentiallyGetSentinelConfig(redis);
+        final RedisConfiguration redisConfiguration = redis.getSentinel() == null
```

Comment:
Use `val` or `var`?

Reply:
ok I change that, but a cast is needed as the inferred type isn't found automatically on next lines as the two configuration implements a commun interface but with a different type.